### PR TITLE
Update readme title and purpose

### PR DESCRIPTION
**Pull request by thephilgray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,8 @@
-# svelte app
+# svelte playground
```
Comment (knoebelja) [inline README.md]: I think this needs proper title formatting. Not sure if you were going for something else on purpose though.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,8 @@
...
-
-```bash
-npm install -g degit # you only need to do this once
-
-degit sveltejs/template-webpack svelte-app
-cd svelte-app
-```
-
-*Note that you will need to have [Node.js](https://nodejs.org) installed.*
+This project was created with the goal of gaining practical experience and insight from working with web components and the Svelte UI framework.
```
Comment (knoebelja) [inline README.md]: We could move the Svelte link here:
>  ... and the [Svelte UI framework.](https://svelte.technology)

File: README.md [hunk 3]
```diff
@@ -1,18 +1,8 @@
...
 
+It was built using a template for [Svelte](https://svelte.technology) apps. The original template lives at https://github.com/sveltejs/template-webpack.
```
Comment (knoebelja) [inline README.md]: Maybe something like this instead:
> It was built using Svelte's [webpack-template.](https://github.com/sveltejs/template-webpack)

File: README.md [hunk 4]
```diff
@@ -1,18 +1,8 @@
...
 
 ## Get started
```
Comment (knoebelja) [inline README.md]: How do you feel about breaking this up into 2 sections: **Initial Setup** and **Development**?

Initial Setup
- clone the repo
- cd into it
- install dependencies

Development
- we recommend `vs code`, npm version, node version, etc.
- use `npm run dev` script

We may eventually also have a Deployment instructions in the future.

**Review by knoebelja — CHANGES_REQUESTED** [review]
Looking good so far. I left some comments that I think could help our documentation at this point in development.

File: README.md (hunk 4 above)
Comment (thephilgray) [reply README.md]: Sounds good. 

Recommending `vs code` is not necessary; editor defaults should be addressed and accounted for with proper linting setup.

We can address node/npm version later when we have a chance to discuss. 

Also, I think we should consider removing existing deployment instructions as part of #9 (Remove serve dependency).

**Comment by thephilgray** [comment]
@knoebelja, Requests addressed

File: README.md [hunk 5]
```diff
@@ -1,36 +1,33 @@
-# svelte app
+# Svelte Playground
 
-This is a project template for [Svelte](https://svelte.technology) apps. It lives at https://github.com/sveltejs/template-webpack.
+This project was created with the goal of gaining practical experience and insight from working with web components and the [Svelte UI framework](https://svelte.technology).
```
Comment (knoebelja) [inline README.md]: Please use a present passive voice instead of past passive.

File: README.md [hunk 6]
```diff
@@ -1,36 +1,33 @@
...
 
-To create a new project based on this template using [degit](https://github.com/Rich-Harris/degit):
+It was built using Svelte's [webpack-template](https://github.com/sveltejs/template-webpack).
```
Comment (knoebelja) [inline README.md]: Present passive again.

File: README.md (hunk 5 above)
Comment (knoebelja) [reply README.md]: Maybe say something like "This project is for gaining...."

File: README.md (hunk 6 above)
Comment (knoebelja) [reply README.md]: "It uses Svelte's..."

File: README.md (hunk 5 above)
Comment (knoebelja) [reply README.md]: Or "The goal of this project is to gain..."

File: README.md [hunk 7]
```diff
@@ -1,36 +1,33 @@
...
 
 ```bash
 npm run dev
 ```
 
-Navigate to [localhost:8080](http://localhost:8080). You should see your app running. Edit a component file in `src`, save it, and the page should reload with your changes.
-
+Navigate to [localhost:8080](http://localhost:8080). You should see the app running. Edit a component file in `src`, save it, and the page should reload with your changes.
 
 ## Deploying to the web
```
Comment (knoebelja) [inline README.md:33]: Add a "To do.." comment maybe?

**Review by knoebelja — CHANGES_REQUESTED** [review]
I just left a few comments. Everything else looks good!

File: README.md (hunk 7 above)
Comment (thephilgray) [reply README.md:33]: Can add when there's a todo.

**Comment by thephilgray** [comment]
@knoebelja, requests addressed

**Review by knoebelja — APPROVED** [review]
LGTM

**Pull request merged**